### ISSUE TITLE
Changing the Browse Order, cont.

### ISSUE THREAD

**Issue by eobrandon99** [CollectionBuilder/collectionbuilder.github.io#98]
### Discussed in https://github.com/orgs/CollectionBuilder/discussions/72

<div type='discussions-op-text'>

<sup>Originally posted by **mldhistory** December 15, 2022</sup>
Hello!
I'm having trouble understanding the implementation of the config-browse.csv file. I would like to modify the sort order on the browse page of my collection so the default order shows the cards in the order that I have them entered in my .csv file. The current default is a random sort.  Also, how would I add additional sort-fields to the drop-down? The current list of sort fields in the dropdown (random,title,date) doesn't seem to correspond to the existing config-browse.csv file entries.
Any advice is much appreciated!
Thanks, Marcie</div>

Is there a way to have the Browse page sort to any other parameter (e.g. Title) by default?

**Comment by evanwill** [comment]
@eobrandon99 yes, but it's not a configuration option so involves editing the template a bit. For example, to sort by Title by default:

In "_layouts/browse.html", you will find text inside a button element that looks like:

```
Sort by <span id="sortFilter">Random</span>
```

Replace the `Random` with `Title`.
Next, just below that look for the buttons that look like:

```
<button class="dropdown-item browse-sort-item active" data-filter="random">Random</button>
<button class="dropdown-item browse-sort-item" data-filter="title">Title</button>
```

Swap the order around like:

```
<button class="dropdown-item browse-sort-item active" data-filter="title">Title</button>
<button class="dropdown-item browse-sort-item" data-filter="random">Random</button>
```

Next in "_includes/js/browse-js.html", search for the lines that look like:

```
/* randomize items once at page load */
shuffle(items);
```

Comment out the shuffle and add a sorting function instead, so it looks like:

```
/* randomize items once at page load */
//shuffle(items);
/* sort by title at page load */
sorting(items,"title");
```

You could do the same with any other metadata field that you want to make default. 
(honestly, this is harder than it should be, we hope to re-write the browse js before too long, and might make this a configuration option or at least easier to do!)
